hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -1381,15 +1391,20 @@ def redistribute_integer_pairs(self, chooser):
             and node1.index < node.index <= node1.index + 4,
         )
 
-        m = node1.value
-        n = node2.value
+        m: Union[int, float] = node1.value
+        n: Union[int, float] = node2.value
 
-        def boost(k):
+        def boost(k: int):
```

Comment:
```suggestion
        def boost(k: int) -> bool:
```

Reply:
thanks - I should add mypy to my prepush hook.